scripts/HPC_chatbot_preprocessor/parsed_mds/generic/connecting/connecting_paragraph_3.txt

```diff
@@ -0,0 +1,9 @@
+Trying to establish an SSH connection from an IP address that does not
+adhere to these restrictions will result in an immediate failure to
+connect, with an error message like:
+ssh_exchange_identification: read: Connection reset by peer
+First Time connection to the HPC infrastructure
+The remaining content in this chapter is primarily focused for people utilizing a terminal with SSH.
```

Comment:
for consitency, and if this truly only for ssh, make a ssh chapter like there is one for the webportal.